torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6736,6 +6827,66 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestCommon', 'test_variant_consistency_jit'),
            ),
            dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16)),
+    OpInfo('norm',
+           backward_dtypesIfCPU=floating_and_complex_types_and(torch.bfloat16),
+           sample_inputs_func=sample_inputs_norm,
+           dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16),
+           skips=(
+               # RuntimeError not raised :
+               # Expected RuntimeError when calling with input.device=cpu and out.device=cuda
+               SkipInfo('TestCommon', 'test_out'),
+           )
+           ),
+    OpInfo('norm',
+           variant_test_name='nuc',
+           sample_inputs_func=partial(sample_inputs_norm, sample_types='nuc'),
+           decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
+           dtypes=floating_and_complex_types(),
+           dtypesIfCUDA=floating_and_complex_types(),
+           skips=(
+               # RuntimeError not raised :
+               # Expected RuntimeError when calling with input.device=cpu and out.device=cuda
+               SkipInfo('TestCommon', 'test_out'),
+               # RuntimeError:
+               # Arguments for call are not valid.
+               SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=(torch.complex64,)),
+               # RuntimeError: aliasOp != torch::jit::getOperatorAliasMap().end()
+               # INTERNAL ASSERT FAILED at "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":157,
+               # please report a bug to PyTorch.
+               SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=(torch.float32,)),
+           )
+           ),
+    OpInfo('norm',
+           variant_test_name='jit',
```

Comment:
As mentioned above this variant seems a little weird. 

Ideally variants would correspond to different code paths with different properties. So if nuclear norm and frobenius norm are actually different functions with different properties then they can have different OpInfos. 

Maybe this can become the frobenius variant?

Reply:
(all of its inputs use the frobenius norm, since the default for norm is frobenius)

Reply:
They have different code-paths.

https://github.com/pytorch/pytorch/blob/44c20ce676c6ca2a8b04af72c80c36698ee34d60/aten/src/ATen/native/cpu/ReduceOpsKernel.cpp#L199-L277